extensions/amp-instagram/0.1/amp-instagram.js

```diff
@@ -85,8 +86,10 @@ class AmpInstagram extends AMP.BaseElement {
     const image = this.win.document.createElement('amp-img');
     // This will redirect to the image URL. By experimentation this is
     // always the same URL that is actually used inside of the embed.
-    image.setAttribute('src', 'https://www.instagram.com/p/' +
-        encodeURIComponent(this.shortcode_) + '/media/?size=l');
+    const imgSrc = resolveUrlAttr('amp-img', 'src',
```

Comment:
The current `createPlaceholderCallback` is supposed to return the element right away. I guess we can create an image and set `src` on it after `whenFirstVisible`. This is also not how `amp-img` works. So, I'm not sure what's the best approach would be here. The easiest option might be to create an `amp-img` as we do now, but set a `prerender=no` attribute on it and teach our runtime to ignore such elements in prerendering mode - this might require the least changes to runtime.